spot_rl_experiments/experiments/skill_test/test_sem_place.py

```diff
@@ -28,24 +35,36 @@
 
     # Start testing
     contnue = True
+    INITIAL_ARM_JOINT_ANGLES = [0, -180, 180, 90, 0, -90]
+    episode_ctr = 0
+    # Get EE Pose Initial
+    spot_pos, spot_ort = spotskillmanager.spot.get_ee_pos_in_body_frame()
+    # Set Orientation as Zero
+    spot_ort = np.zeros(3)
     while contnue:
+        # Open Gripper
+        spotskillmanager.spot.open_gripper()
+        input("Place an object in Spot's gripper and press Enter to continue...")
+        # Place Object and Close Gripper
         rospy.set_param("is_gripper_blocked", 0)
-        spotskillmanager.place(place_target, is_local=is_local, visualize=True)
-        contnue = map_user_input_to_boolean("Do you want to do it again ? Y/N ")
+        episode_log = {"actions": []}
+        spotskillmanager.spot.close_gripper()
+        input("waiting for user to get ready with camera")
 
-# The following is a helpful tip to debug the arm
-# We get Spot class
-# spot = spotskillmanager.spot
-# We can move the gripper to a point with x,y,z and roll, pitch, yaw
-# spot.move_gripper_to_point((0.55, 0., 0.26), np.deg2rad(np.array([0,0,0])))
-# We can also set the robot arm joints
-# config = construct_config()
-# spot.set_arm_joint_positions(np.deg2rad(config.INITIAL_ARM_JOINT_ANGLES))
-
-# In addition, if you want to use semantic place skill based on the grasping orientation, you can do
-# spotskillmanager.nav("black_case")
-# spotskillmanager.pick("bottle")
-# # Fetch the arm joint at grasping location
-# ee_orientation_at_grasping = spotskillmanager.gaze_controller.env.ee_orientation_at_grasping
-# spotskillmanager.nav("test_desk")
-# spotskillmanager.place("test_desk", orientation_at_grasping) # This controls the arm initial orientation
+        spotskillmanager.place(place_target, is_local=is_local, visualize=False)
```

Comment:
please add the comments from the original script back as it contains useful information to debug